From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Tue, 13 May 2025 14:41:06 +0000
Subject: [PATCH] Increment maximum number of search engines

Commands used for incrementing version:

```bash
    version_line=$(echo $(grep kCurrentDataVersion definitions/prepopulated_engines.json))

    version=$(echo $version_line | sed -r "s/.*\"kMaxPrepopulatedEngineID\": //g")

    incremented_version=$(($version + 1)); incremented_version_line=$(echo $version_line | sed -r "s/$version/$incremented_version/g")

    sed -ri "s/$version_line/$incremented_version_line/g" definitions/prepopulated_engines.json

```
---
 definitions/prepopulated_engines.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/definitions/prepopulated_engines.json b/definitions/prepopulated_engines.json
index c899bcb..80873cb 100644
--- a/definitions/prepopulated_engines.json
+++ b/definitions/prepopulated_engines.json
@@ -22,7 +22,7 @@
     // use a lower number. This number is used by UMA stats and must be updated
     // to include the full range or else not all engines will be counted in
     // stats.
-    "kMaxPrepopulatedEngineID": 116,
+    "kMaxPrepopulatedEngineID": 117,
 
     // Increment this if you change the data in ways that mean users with
     // existing data should get a new version. Otherwise, existing data may
